TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/DriveVelocityPIDTuner.java

```diff
@@ -30,11 +30,20 @@
  * loop velocity control. Although it may seem unnecessary, tuning these coefficients is just as
  * important as the positional parameters. Like the other manual tuning routines, this op mode
  * relies heavily upon the dashboard. To access the dashboard, connect your computer to the RC's
- * WiFi network and navigate to https://192.168.49.1:8080/dash in your browser. Once you've
- * successfully connected, start the program, and your robot will begin moving forward and backward
- * according to a motion profile. Your job is to graph the velocity errors over time and adjust the
- * PID coefficients. Once you've found a satisfactory set of gains, add them to your drive class
- * ctor.
+ * WiFi network. In your browser, navigate to https://192.168.49.1:8080/dash if you're using the RC
+ * phone or https://192.168.43.1:8080/dash if you are using the Control Hub. Once you've successfully
+ * connected, start the program, and your robot will begin moving forward and backward according to
+ * a motion profile. Your job is to graph the velocity errors over time and adjust the PID
+ * coefficients (note: the tuning variable will not appear until the op mode finishs initializing).
```

Comment:
nit: finishes instead of finishs

Reply:
I actually copy pasted that line from the Road Runner documentation. So the same spelling error is there too. I didn't even notice.

![Screen Shot 2020-09-27 at 6 49 22 PM](https://user-images.githubusercontent.com/6739076/94378874-3965f180-00f2-11eb-87d9-4c5ced8615b6.png)